floris/tools/optimization/yaw_optimization/yaw_optimization_base.py

```diff
@@ -264,7 +259,7 @@ def _reduce_control_problem(self):
 
         # Initialize subset variables as full set
         self.fi_subset = self.fi.copy()
-        nwinddirections_subset = copy.deepcopy(self.fi.floris.flow_field.n_wind_directions)
+        n_wind_directions_subset = copy.deepcopy(self.fi.floris.flow_field.n_findex)
```

Comment:
Ah, now I see, you have it working, maybe ignore comment above if all is fine?

Reply:
I think it could use some more testing to make sure it is working 100% as it should.